app/models/energy/construction.py

```diff
@@ -428,7 +484,7 @@ class EnergyWindowMaterialBlind(BaseModel):
         ' Default value is 0.5.”'
     )
 
-    infrared_hemispherical_transmittance: float = Schema(
+    infrared_hemispherical_transmittance: float=Schema(
```

Comment:
I don't think we need the word `hemispherical` in the name here.  Let's just call it `infrared_transmittance` and move the "hemispherical" to the input description.